template.tpl

```diff
@@ -104,99 +104,83 @@ ___TEMPLATE_PARAMETERS___
     "type": "SELECT"
   },
   {
+    "help": "Choose a Google Tag Manager variable that returns an object of key-value pairs that will then be encoded as the parameters for this hit. Select \"No\" if you want to add the parameters manually.",
+    "macrosInSelect": true,
+    "selectItems": [
+      {
+        "displayValue": "No",
+        "value": "no"
+      }
+    ],
+    "displayName": "Retrieve Parameters From Variable",
+    "simpleValueType": true,
+    "name": "paramsFromVariable",
+    "type": "SELECT",
     "enablingConditions": [
       {
         "paramName": "eventType",
-        "type": "EQUALS",
-        "paramValue": "trackStructEvent"
+        "paramValue": "adTracking",
+        "type": "EQUALS"
       },
       {
         "paramName": "eventType",
-        "type": "EQUALS",
-        "paramValue": "trackSelfDescribingEvent"
+        "paramValue": "cartTracking",
+        "type": "EQUALS"
       },
       {
         "paramName": "eventType",
-        "type": "EQUALS",
-        "paramValue": "trackSocialInteraction"
+        "paramValue": "trackConsent",
```

Comment:
Think Consent ignores paramObj still so should probably not be enabled there unless you plan to add support for that.

Reply:
Removed!